operator/pkg/controllers/configconnectorcontext/configconnectorcontext_controller_test.go

```diff
@@ -573,7 +573,7 @@ func TestHandleReconcileFailed(t *testing.T) {
 		t.Errorf("error handling failed reconciliation: %v", err)
 	}
 
-	expectedErrMsg := fmt.Sprintf(k8s.ReconcileErrMsgTmpl, reconcileErr)
+	expectedErrMsg := fmt.Errorf(k8s.ReconcileErrMsgTmpl, reconcileErr).Error()
```

Comment:
Maybe we make this one a hard-coded string (and the other test)?  So then we actually check that the value is reasonable...  Otherwise it feels like we're testing the go compiler.